lib/Client.js

```diff
@@ -1423,6 +1423,30 @@ class Client extends EventEmitter {
         }).then((guild) => new Guild(guild, this));
     }
 
+    /**
+    * Edit guild channels' positions. Note that channel position numbers are lowest on top and highest at the bottom.
+    * @arg {String} guildID The ID of the guild
+    * @arg {Array<Object>} channelPositions An array of [ChannelPosition] (https://discord.com/developers/docs/resources/guild#modify-guild-channel-positions)
+    * @arg {String} ChannelPosition[].id The ID of the channel
+    * @arg {Number} ChannelPosition[].position The new position of the channel
+    * @arg {Boolean} [ChannelPosition[].lockPermissions] Whether to sync the permissions with the new parent if moving to a new category
+    * @arg {String} [ChannelPosition[].parentID] The new parent ID (category channel) for the channel that is moved (only one channel can change category per request)
```

Comment:
what's with the PascalCase and the mismatch param name

Reply:
I just copied this formatting from somewhere else in the code. How should it be?

Reply:
Should always match the name of the parameter, like so:

```suggestion
    * @arg {String} [channelPositions[].parentID] The new parent ID (category channel) for the channel that is moved (only one channel can change category per request)
```

See the `activities` parameter of [`editStatus`](https://abal.moe/Eris/docs/0.16.1/Client#method-editStatus) for documenting an array of objects.

Reply:
I just pushed the corrections. Hope it is ok now. Thanks for all the help and feedback